src/main/java/PDLayer.mdl

```diff
@@ -601,3 +609,28 @@ type PDSigRef extends PDObject {
 	% true if the document permissions dictionary contains DocMDP entry
 	property permsContainDocMDP: Boolean;
 }
+
+% PDF function type
+type PDFunction extends PDObject {
+	% type of the function (0, 2, 3 or 4)
+	property FunctionType: Integer;
+}
+
+% Type0 function
+type PDType0Function extends PDFunction {
+}
+
+% Type2 function
+type PDType2Function extends PDFunction {
+}
+
+% Type3 function
+type PDType3Function extends PDFunction {
+	link Functions: PDFunction*;
```

Comment:
previously all link names started from lower case. Is it correct to use here an Upper case?

Reply:
It is correct, because we leave the upper case if this is the key from the PDF specification